### DISCUSSION TITLE
How to ensure a subclassed Quantity instance always renders trailing zeros to a given degree of precision

### DISCUSSION THREAD

**Discussion by jeffwright13** [KenKundert/quantiphy#29] [Q&A] [answered]
I have a subclass based on the Quantity class

```
class Watt(Quantity):
    units = "W"
    prec = 2
```

If the value is 0, `log` or `print` statements only render `0 W`, not `0.00 W`. I'd like to have those trailing digits show, so text tables with the rendered Quantity values line up neatly. 

Any suggestions?

Thanks, JW

**Comment by KenKundert** [comment, marked as answer]
The first example in [Subclasing Quantities](https://quantiphy.readthedocs.io/en/stable/user.html#subclassing-quantity) gives an example of how to do this:
```
>>> class Watt(Quantity):
...     units = 'W'
...     prec = 2
...     strip_zeros = False

... pwr=Watt(0)
... print(pwr)
0.00 W
```

If you are formatting quantities for output in tables, you might want to also take a look at [Formatting Tabular Data](https://quantiphy.readthedocs.io/en/stable/user.html#formatting-tabular-data).

**Reply by jeffwright13** [reply]
thank you
